Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM python:3.5-alpine
+FROM python:3.7-alpine3.10
```

Comment:
Can this be `alpine:latest`?

Reply:
yeah - updated (also re-scanned to verify trivy is still happy)